### DISCUSSION TITLE
how to switch iframe in HTML

### DISCUSSION THREAD

**Discussion by wpy1215** [AutomaApp/automa#1643] [Q&A]
by using SELEN we could switch frames between frame and main by "driver.switch_to.frame(0)",
 but how to switch frames by using AUTOMA ?

**Comment by Kholid060** [comment]
Use switch frame block
https://docs.automa.site/blocks/switch-to.html#switch-frame-block

**Reply by wpy1215** [reply]
thanks, switched sucessed, but I found that a "timepicker" in webpage which there are some events were added to a element in HTML code, and how to activate there events to call out the "timepicker" ?
